packages/cascara/src/private/ViewConfig/ViewConfigItem.js

```diff
@@ -16,6 +16,11 @@ const propTypes = {
   label: pt.string.isRequired,
   moveItemDown: pt.func.isRequired,
   moveItemUp: pt.func.isRequired,
+  option: pt.shape({
+    attribute: pt.string,
+    module: pt.string,
+    required: false,
+  }),
```

Comment:
The original option shape could be anything. We are going to cause ourselves proptype errors if we try to enforce properties inside the shape.